site/src/components/sampler/largestTriangleOne.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: component
+title: Largest Triangle 1 Bucket
+component: data/sampler/largestTriangleOneBucket.js
+namespace: samplers
+
+example-code: |
+    // create some random data
+    var data = fc.data.random.financial()(1000);
+
+    // configure the sampler
+    var sampler = fc.data.sampler.largestTriangleOneBucket()
+        .bucketSize(20)
+        .xValue(function(d) { return d.date; })
+        .yValue(function(d) { return d.low; });
+
+    // sample the data
+    var sampledData = sampler(data);
+
+    xScale.domain(fc.util.extent().fields('date')(sampledData))
+    yScale.domain(fc.util.extent().fields('low')(sampledData));
+
+    // render the sampled data
+    var sampledLine = fc.series.line()
+        .xScale(xScale)
+        .yScale(yScale)
+        .yValue(function (d) { return d.low; });
+
+    container.append('g')
+        .datum(sampledData)
+        .call(sampledLine);
+
+    // render the original data
+    var originalLine = fc.series.line()
+        .xScale(xScale)
+        .yScale(yScale)
+        .yValue(function (d) { return d.low; })
+        .decorate(function(selection) {
+            selection.enter().style('stroke-opacity', '0.5');
+        });
+
+    container.append('g')
+        .datum(data)
+        .call(originalLine);
+---
+
+The Largest Triangle One Bucket component subsamples data by calculating the largest areas between a data point and its immediate neighbours.
+
+The sampler requires both the `xValue` and `yValue` properties in order to generate calculate the area. You can configure the sampling frequency by setting the `bucketSize` property.
```

Comment:
If you're calculating area do you not need to use the scaled values? There's no guarantee that the values themselves have any linear relationship with the scaled values.